backend/src/modules/idim-webservice/idim-webservice.controller.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    Controller,
+    Get,
+    Query,
+    HttpException,
+    HttpStatus,
+    UseGuards,
+} from '@nestjs/common';
+import { ApiResponse, ApiQuery, ApiTags, ApiBearerAuth } from '@nestjs/swagger';
+import { AuthGuard } from '../auth/auth.guard';
+import { IdimWebserviceService } from './idim-webservice.service';
+import {
+    IDIRUserResponse,
+    RequesterAccountTypeCode,
+} from './idim-webservice.dto';
+
+@ApiTags('IDIM Webservice')
+@UseGuards(AuthGuard)
+@ApiBearerAuth('bearerAuth')
+@Controller('idim-webservice')
+export class IdimWebserviceController {
+    constructor(
+        private readonly idimWebserviceService: IdimWebserviceService
+    ) {}
+
+    @Get('idir')
+    @ApiResponse({ status: HttpStatus.OK, type: IDIRUserResponse })
+    @ApiQuery({
+        name: 'requesterAccountTypeCode',
+        enum: RequesterAccountTypeCode,
+    })
+    async verifyIdirUser(
+        @Query('userId') userId: string,
+        @Query('requesterUserId') requesterUserId: string,
+        @Query('requesterAccountTypeCode')
+        requesterAccountTypeCode: string
```

Comment:
Hi Ian, I wonder if there is a way to set a default value for the this parameter? I think most of the case it will be IDIR user account login to FAM and grant access.